config/locales/forms/2023/lettings/property_information.en.yml

```diff
@@ -6,99 +6,117 @@ en:
           uprn:
             page_header: ""
             uprn_known:
-              check_answer_label: "UPRN known?"
+              check_answer_label: "UPRN known"
+              check_answer_prompt: "Enter UPRN if known"
               hint_text: "The Unique Property Reference Number (UPRN) is a unique number system created by Ordnance Survey and used by housing providers and various industries across the UK. An example UPRN is 10010457355.<br><br>The UPRN may not be the same as the property reference assigned by your organisation.<br><br>If you don’t know the UPRN you can enter the address of the property instead on the next screen."
               question_text: "Do you know the property's UPRN?"
             uprn:
               check_answer_label: "UPRN"
+              check_answer_prompt: ""
               hint_text: ""
               question_text: "What is the property's UPRN?"
 
           uprn_confirmed:
             page_header: "We found an address that might be this property"
             check_answer_label: "Is this the right address?"
+            check_answer_prompt: "Answer if this is the right address"
             hint_text: ""
             question_text: "Is this the property address?"
 
           address:
             page_header: "Q12 - What is the property's address?"
             address_line1:
               check_answer_label: "Address lines 1 and 2"
+              check_answer_prompt: "Enter address lines 1 and 2"
               hint_text: ""
               question_text: "Address line 1"
             address_line2:
               check_answer_label: ""
+              check_answer_prompt: ""
               hint_text: ""
               question_text: "Address line 2 (optional)"
             town_or_city:
               check_answer_label: "Town or city"
+              check_answer_prompt: ""
               hint_text: ""
               question_text: "Town or city"
             county:
               check_answer_label: "County"
+              check_answer_prompt: ""
               hint_text: ""
               question_text: "County (optional)"
             postcode_full:
               check_answer_label: "Postcode"
+              check_answer_prompt: ""
               hint_text: ""
               question_text: "Postcode"
 
           la:
             page_header: ""
             check_answer_label: "Local authority"
+            check_answer_prompt: ""
             hint_text: ""
             question_text: "What is the property’s local authority?"
 
           first_time_property_let_as_social_housing:
             page_header: ""
-            check_answer_label: "First time being let as social-housing?"
+            check_answer_label: "First time being let as social-housing"
+            check_answer_prompt: "Answer if it's the first time being let as social-housing"
             hint_text: ""
```

Comment:
The `check_answer_prompt` text is subject to change after review by the content designer, as are some of the `check_answer_label` texts. Following my last discussion with the content designer, we are going to rewrite the check answer labels to avoid needing a question mark at the end.